webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Parameters/RowActions.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Td, ActionsColumn } from '@patternfly/react-table';
+import { APIActions } from '../../../../redux/API';
+import { sprintf, translate as __ } from '../../../../common/I18n';
+import { HOST_PARAM } from './ParametersConstants';
+import { openConfirmModal } from '../../../ConfirmModal';
+import { updateHost } from '../../ActionsBar/actions';
+
+export const RowActions = ({ hostId, param }) => {
+  const dispatch = useDispatch();
+  const onDelete = () =>
+    dispatch(
+      APIActions.delete({
+        url: `/api/hosts/${hostId}/parameters/${param.id}`,
+        key: `${param.id}-DELETE`,
+        successToast: () =>
+          sprintf(__('Parameter %s has been deleted successfully'), param.name),
+
+        errorToast: ({ response }) =>
+          // eslint-disable-next-line camelcase
+          response?.data?.error?.full_messages ||
+          response?.data?.error?.message,
+        handleSuccess: () => {
+          dispatch(updateHost(hostId));
+        },
+      })
+    );
+  const rowActions = [
+    param.type === HOST_PARAM && {
+      title: __('Delete'),
+      onClick: () => {
+        dispatch(
+          openConfirmModal({
+            title: `${__('Delete ')} ${param.name}`,
```

Comment:
This should use `sprintf` like on line 19.

Reply:
@MariaAga this is still unaddressed